### PR TITLE
Feature/adding back button to all chart pages

### PR DESCRIPTION
**Pull request by Dinisueldanov**
### Summary
- added back button to all chart pages

### Screenshots
statistics-all-charts-back-button

### CODE REVIEW

File: src/features/pastelStatistics/components/chart/EChartsLineChart.tsx [hunk 1]
```diff
@@ -164,6 +166,9 @@ export const EChartsLineChart = (props: TLineChartProps): JSX.Element => {
   return (
     <div className={styles.container}>
       <div className={styles.lineChartHeader}>
+        <Link to={routes.STATISTICS} className={styles.backButtonLink}>
+          <span className={styles.arrowBackButton}></span>
+        </Link>
```
Comment (ngvtuan) [inline src/features/pastelStatistics/components/chart/EChartsLineChart.tsx:171]: can we use the fontawesome icon for the back button? Because we already added/used `"@fortawesome/fontawesome-free": "5.12.0"` in the wallet. (see the sidebar) https://fontawesome.com/v5.15/icons/arrow-circle-left?style=solid

And I think we can create a component and re-use it instead of adding the same code for every chart.

File: src/features/pastelDB/pastelDBLib.ts [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import { remote } from 'electron'
 import fs from 'fs'
+import { open } from 'fs/promises'
```
Comment (ngvtuan) [inline src/features/pastelDB/pastelDBLib.ts:3]: I think the change in this file does not belong to this PR. Can we remove un-needed code?

**Review by sanjanadhikari1 — APPROVED** [review]
Working fine and looks good.

**Review by ngvtuan — APPROVED** [review]

**Review by sanjanadhikari1 — APPROVED** [review]
